include/mesh/boundary_info.h

```diff
@@ -115,7 +115,7 @@ class BoundaryInfo : public ParallelObject
    * pass a pointer to both the boundary_mesh's MeshData object,
    * and the MeshData object used for this mesh.
    */
-  void sync (const std::set<boundary_id_type> &requested_boundary_ids,
+  void sync (const std::set<boundary_id_type>& requested_boundary_ids,
```

Comment:
> Are we getting OCD about & and * placement now? Can I put in a vote for "separate it by spaces on both sides in declarations"? I was surprised at how much more readable that style made MOOSE code.

Absolutely, "space on both sides" is now my preference as well.  I changed these ones because the original libmesh convention was always "space on the right" and the rest of the file used that convention except for a few lines...